documentation/puppet-api/v3/compile.markdown

```diff
@@ -36,7 +36,8 @@ The request body must look like:
   "transaction_uuid": "<uuid string>",
   "job_id": "<id string>",
   "options": { "capture_logs": <true/false>,
-               "log_level": <one of debug/info/warning/err>}
+               "log_level": <one of debug/info/warning/err>
+               "bolt": <true/false> Whether or not to attempt to load bolt }
```

Comment:
What if we called this `load_bolt` so that we can drop the explanation? Just noticing that the other options don't have that.

Reply:
I forgot to push my nitpick in the last PR: personally I think this should be something like `load_bolt_types` or similar to identify _why_ you would need to load bolt.

Reply:
Bolt https://github.com/puppetlabs/bolt/blob/master/lib/bolt/applicator.rb#L94 adds a fact for compilation indicating it is being done in a special "bolty" way. I ported that behavior to orch https://github.com/puppetlabs/orchestrator/pull/1340/files#diff-e3cb1255de6ee2d1bf0cf60785425dd0R305 . I named this accordingly. Perhaps it is a bolt centric way of thinking about it.

Reply:
`compile_for_plan` maybe then?

Reply:
Ok it will be trivial to update. I just has to happen in two projects. We can bike shed on it for a bit and i'll update it to whatever we come up with.